### PR TITLE
coala-header.svg: Fix coala.io text rendering

### PR DESCRIPTION
**Pull request by sangamcse**
This combines `coala-color.svg` and `coala-text.svg` using
`<image>` SVG element which includes images inside SVG
document. It takes image links generated from GitHub in the
`href` of `<image>` tag.

Closes https://github.com/coala/artwork/issues/53

### CODE REVIEW

**Review by ayan-b — APPROVED** [review]
LGTM.

**Review by jayvdb — CHANGES_REQUESTED** [review]
The diff is unreadable.

The actual content changes cant be seen.

Split this into two commits, one which changes the style, and the other which changes the content.

**Comment by sangamcse** [comment]
@jayvdb TBH, these changes in SVG XML format is done by Inkscape which I tried to reduce. My only concern was the logo text rendering which was a bit odd in the previous version but this change looks little satisfying to my eyes. 😄

File: artwork/logo/coala-header.svg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!-- Created with Inkscape (http://www.inkscape.org/) -->
 
-<svg
+<svg version="1.1"
```
Comment (jayvdb) [inline artwork/logo/coala-header.svg:3]: moving version isnt part of "fix coala.io text..."

File: artwork/logo/coala-header.svg [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
    xmlns:xlink="http://www.w3.org/1999/xlink"
-   x="0px"
-   y="0px"
-   viewBox="0 0 193.2 41.1"
-   xml:space="preserve">
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   viewBox="0 0 193.2 41.099998"
+   version="1.1"
+   id="svg3781"
```
Comment (jayvdb) [inline artwork/logo/coala-header.svg]: the old `id` value was better

File: artwork/logo/coala-header.svg [hunk 3]
```diff
@@ -10,28 +10,89 @@
    xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   viewBox="0 0 193.2 41.099998"
-   version="1.1"
+   x="0px"
+   y="0px"
+   viewBox="0 0 193.2 41.1"
```
Comment (jayvdb) [inline artwork/logo/coala-header.svg]: these changes should be done in the other commit.

I doubt this is part of fixing coala.io, but it is a sensible change as part of fixing the file structure (the other commit).
```diff
-viewBox="0 0 193.2 41.099998"
+viewBox="0 0 193.2 41.1
```

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by jayvdb** [comment]
the commit message of https://github.com/coala/artwork/pull/54/commits/ead8a4c83d6 is a bit odd.

"betterSVG."

ALso `version="1.1"` is moved twice in these two patches - returning to the same spot in the second patch.

Can we use https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image to put 'coala.io' text in a separate SVG  file which is re-usable , and also to re-use the real coala logo svg instead of creating a copy?

It would be better to try using https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text for the 'coala.io'

**Comment by sangamcse** [comment]
@jayvdb, I agree. Tried creating one at https://github.com/sangamcse/artwork/blob/coala-text/artwork/logo/coala_io_text.svg. Is it ok?
> Can we use https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image to put 'coala.io' text in a separate SVG file which is re-usable, and also to re-use the real coala logo SVG instead of creating a copy?

**Comment by jayvdb** [comment]
The o in `.io` isnt rendering for me in Falkon.

@ayan-b can you check?

**Comment by sangamcse** [comment]
> Can we use https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image to put 'coala.io' text in a separate SVG file which is re-usable , and also to re-use the real coala logo svg instead of creating a copy?

Created an issue for separate `coala.io` text. https://github.com/coala/artwork/issues/56

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ack https://github.com/coala/artwork/pull/54/commits/e6575798b824c2318d4382c9a73730876ef45c63

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**